package.json

```diff
@@ -3,25 +3,31 @@
   "version": "0.3.1",
   "scripts": {
     "build": "rollup -c",
+    "dev": "rollup -c --config-enable-developer-mode",
     "lint": "eslint . && web-ext lint",
     "package": "npm run build && web-ext build --overwrite-dest --ignore-files \"./tests/**/*\" --ignore-files \"./src/**/*\" --ignore-files \"./rollup.config.js\"",
     "start": "npm run build && web-ext run",
-    "test-integration": "npm run package && mv web-ext-artifacts/*.zip web-ext-artifacts/study.xpi && mocha --timeout 30000 \"./tests/integration/*.js\""
+    "test-integration": "npm run package && mv web-ext-artifacts/*.zip web-ext-artifacts/study.xpi && mocha --timeout 30000 \"./tests/integration/*.js\"",
+    "watch": "npm-run-all --parallel watch-addon watch-webext",
+    "watch-addon": "npm run dev -w",
+    "watch-webext": "web-ext run --watch-file dist/background.js"
```

Comment:
Nit, but I am confused about the distinction between `webext` and `addon` here :) The usual way these terms are used is that a Add-ons are an umbrella term that encompasses both WebExtensions and Themes (and formerly legacy Plug-ins, these are all deprecated except for the DRM stuff but you can still see them in `about:addons`)

Reply:
Thanks for the clarification. I renamed the scripts. Is this better?